modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -63,7 +65,27 @@ Error handler after API flow:
 
 image::apim-message-logging-policy-states-3.png[State after the error handler executes]
 
-== Logging Policy in XML
+
+== Configuring Policy Parameters
+
+When you apply the Message Logging policy to your API from the UI, you can configure the following parameters:
+
+image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description | Required
+|Message |A DataWeave expression to use for extracting the message to be logged.
+Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).|Required
+|Conditional |A DataWeave Boolean expression used to specify whether a message is to be logged. |Optional.
```

Comment:
No periods at the end of fragment descriptions
```suggestion
|Conditional |A DataWeave Boolean expression used to specify whether a message is to be logged |Optional
```

Reply:
Done! Thanks!